lib/persisted-model.js

```diff
@@ -1176,7 +1182,9 @@ module.exports = function(registry) {
     async.waterfall(tasks, done);
 
     function getSourceChanges(cb) {
-      sourceModel.changes(since.source, options.filter, debug.enabled ? log : cb);
+      utils.downloadInChunks(options.filter, chunkSize, function(filter, pagingCallback) {
```

Comment:
When function arguments don't fit on a single line, then we are applying "one arg per line" rule, see http://loopback.io/doc/en/contrib/style-guide.html#one-argument-per-line

```js
utils.downloadInChunks(
  options.filter, chunkSize, 
  function(filter, pagingCallback) {
    sourceModel.changes(since.source, filter, pagingCallback);
  },
  debug.enabled ? log : cb);
```  

This applies to all other `utils.*InChunks` calls below too.